### ISSUE TITLE
Gesture configuration aka save aka settings file path

### ISSUE THREAD

**Issue by ivribalko** [rxhanson/Multitouch-Community#135]
Hello Ryan! 

Thank you for the amazing app!

I would like to know where the configuration file for the gestures is stored.

I'd like to back it up manually and transfer between different computers. I can't quite use iCloud because I have different accounts on those computers and I don't think there's a way to download the config from https://www.icloud.com?

**Comment by rxhanson** [comment]
Here's a short wiki page I made about this:
https://github.com/rxhanson/Multitouch-Community/wiki/Transferring-Multitouch-Settings-to-a-Different-Mac

Let me know if you have any issues.

**Comment by ivribalko** [comment]
Thanks! 

I think changing `~/Library/Preferences/com.brassmonkery.Multitouch.plist` only works when the app's not installed. 

If I quit `Multitouch` and make a change to the settings file (or remove it altogether) and restart the app - all my settings are preserved in the app as if I did no changes even though I can see that the contents of the settings file are different from before. **Settings are preserved** even if I remove the app before removing the settings file and re-install it afterwards.

And if I place the settings file to the right place beforehand and install `Multitouch` onto a machine that **never had it installed** then settings are correctly read and applied.

Is there a settings cache somewhere that I could purge? I've tried to `find /Users/ -iname "*multitouch*" 2>/dev/null` everything but so far was not able to get it to read the changed settings file.

**Comment by ivribalko** [comment]
OK, I've figured out couple more things. Like it says it only stores **gestures** in iCloud. Also manual settings file changes get applied only after computer restart (restarting the app is not enough). So in order to successfully restore all settings including gestures what I do is:
- make sure settings file has `Sync gesture config in iCloud` disabled before backing it up (otherwise it'll pull latest gestures from iCloud upon app start instead of reading them from the restored settings file and will then overwrite it)
- quit Multitouch with command+Q
- replace settings file
- restart computer

I wonder if any of my findings deserve a spot in [the manual](https://github.com/rxhanson/Multitouch-Community/wiki/Transferring-Multitouch-Settings-to-a-Different-Mac). Took me a good while to figure this one out. Closing this now.

Thank you!
